pkg/web/api/pipeline_v1.go

```diff
@@ -124,6 +124,9 @@ func (p *PipelineAPIv1) CreatePipeline(
 	if err != nil {
 		return nil, status.PipelineError(cerrors.Errorf("failed to create pipeline: %w", err))
 	}
+	if len(created.Config.Description) > 250 {
```

Comment:
250 sounds too low for the pipeline description IMO. Even our very simple [example pipeline](https://github.com/ConduitIO/conduit/blob/7cd6af09fa8f15a3cd6665a37034522b6fe33b03/examples/pipelines/file-to-file.yml#L6-L9) has a description length of 237 characters. I see the description as a field where you can describe what the pipeline is doing, for complex pipelines this could get quite long. I'd maybe go with a limit of 8192 (2^13), this should give plenty of space to the user to describe the pipeline while giving us the confidence that the description won't consume more than 8kB.

WDYT @hariso @maha-hajja @jmar910 @ahamidi? I know that longer descriptions might look bad in the UI right now, but I think we could encounter them in pipeline configuration files, the UI can always be adjusted.

Reply:
I think the 8192 limit sounds fair, not too long yet long enough to explain a somewhat complicated pipeline.